src/CSH/Eval/Frontend.hs

```diff
@@ -63,8 +54,7 @@ getEvalAPI _ = WaiSubsite evalAPI
 
 -- | An example Html handler for the frontend. 
 getHomeR :: Handler Html
-getHomeR = defaultLayout $(whamletFile "frontend/index.hamlet")
```

Comment:
`defaultLayout` is set above to `evalLayout`. Even thought the two definitions are the same, I would say we should keep it as `defaultLayout`.